sys/include/net/gnrc/netif/conf.h

```diff
@@ -101,6 +101,18 @@ extern "C" {
 #define GNRC_NETIF_IPV6_RTR_ADDR   (0)
 #endif
 
+/**
+ * @brief   Number of global unicast addresses needed for a
+ *          @ref net_dhcpv6_client "DHCPv6 Client" using IA_NA
+ *
+ * @note    Used for calculation of @ref CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF
+ */
+#ifdef MODULE_GNRC_DHCPV6_CLIENT_IA_NA
+#define GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR   (1)
```

Comment:
Should a `NUMOF` suffix be added to this identifier?

Reply:
Yes. And for brevity maybe use

```suggestion
#define GNRC_NETIF_IPV6_DHCPV6_IA_NA_NUMOF  (1)
```

instead. The NA stands for non-temporary _address_ after all, so the `ADDR` is somewhat redundant ;-).

Reply:
I renamed it 44ed30a0d1840a03041ad62e9073e598b1972eab :)